app/aboot/lk2nd-device.h

```diff
@@ -10,6 +10,12 @@ struct lk2nd_device {
 	const char *cmdline;
 	const char *bootloader;
 	const char *model;
+
+	// Properties used for motorola unit info
```

Comment:
now it's time to remove this comment and to move these properties directly below bootloader. This is not supposed to be too Motorola-specific.